docs/source/testing/mocking.md

```diff
@@ -30,15 +30,19 @@ server.listen().then(({ url }) => {
 });
 ```
 
-> Note: If `typeDefs` has custom scalar types, `resolvers` must still contain the `serialize`, `parseValue`, and `parseLiteral` functions
+> Note: If `typeDefs` has any custom scalar types, your `resolvers` must contain the `serialize`, `parseValue`, and `parseLiteral` functions for each of those custom scalars.
 
-Mocking logic simply looks at the type definitions and returns a string where a string is expected, a number for a number, etc. This provides the right shape of result. By default, when using mocks, any existing resolvers are ignored. See the ["Using existing resolvers with mocks"](#using-existing-resolvers-with-mocks) section below for more info on how to change this behavior.
+Mocking logic looks at the types expected by your schema's fields and returns values based on those types. For example, mocking will return a string where a string is expected, a number for a Int, etc. The shape of your mocked results should always be consistent with the shape of a real result from a successful GraphQL operation.
 
-For more sophisticated testing, mocks can be customized to a particular data model.
+When using mocks your existing resolvers are **ignored** by default. See the ["Using existing resolvers with mocks"](#using-existing-resolvers-with-mocks) section below to see how to change this behavior.
+
+For more sophisticated testing, you can customize your mocks to return user-specified data.
 
 ## Customizing mocks
 
-In addition to a boolean, `mocks` can be an object that describes custom mocking logic, which is structured similarly to `resolvers` with a few extra features aimed at mocking. Namely `mocks` accepts functions for specific types in the schema that are called when that type is expected. By default, the functions in `mocks` will overwrite the resolvers in `resolvers`. In this example `hello` and `resolved` will both return `'Hello'`.
+The `mocks` option can be assigned to an object describing custom mocking logic, allowing you to specify the values to return for specific field types. The `mocks` object allows you to define functions for field types, which will be invoked when that field type is expected. 
```

Comment:
I think I might suggest something like:

"Instead of providing a boolean to the `mocks` option, you can provide an object that defines custom mocking logic. This enables you to specify values to return for different return types."